qiskit/circuit/__init__.py

```diff
@@ -262,9 +262,9 @@
 .. autosummary::
    :toctree: ../stubs/
 
-    Parameter
-    ParameterVector
-    ParameterExpression
+   Parameter
+   ParameterVector
+   ParameterExpression
```

Comment:
Are these indentation changes intentional? 🙂

Reply:
Yes. After I added `abs()` method, I tried `tox -edocs` to create a document, but I could not find `abs()` method in the ParameterExpression document. According to [this stack overflow](https://stackoverflow.com/questions/47715781/how-to-add-a-toctree-entry), the numbers of whitespaces have to be the same for all entries, so I changed the indentation and this solved the problem of missing `abs()` method in the document.